pyproject.toml

```
@@ -2,6 +2,135 @@
 requires = ["setuptools", "wheel", "setuptools-rust"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "qiskit"
+description = "An open-source SDK for working with quantum computers at the level of extended quantum circuits, operators, and primitives."
+requires-python = ">=3.8"
+license = { file = "LICENSE.txt" }
```

Should we add?
```suggestion
license = { file = "LICENSE.txt",  text = "Apache 2.0" }
```

I'm not sure if this is valid syntax or not though. I was doing one final pass through comparing the old setup.py with the metadata in the `pyproject.toml` and noticed we had the license string set before (and the file was pulled in implicitly I think) but now we just point to the file. But I also need to read PEP 639 to see if this matters at all either.

---

[The current PyPA guidelines say it's either-or](https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license) (though maybe others interpret that text differently), so maybe it's best to just revisit when PEP 639 is accepted?

---

Yeah, we can look at it again if/when pep 639 is accepted.